test/helpers/kafka_conf.go

```diff
@@ -0,0 +1,298 @@
+package helpers
```

Comment:
If we need to make these specific to the functional framework they may need to be added into test/functional  It depends if there is network info that makes specific assumptions about the containers and how the servers are found